src/commands/tick-log.js

```diff
@@ -40,10 +43,12 @@ function log(argv) {
   let { start, end } = parseDateRange(argv.date)
   start = moment(start).toArray()
   end = moment(end).toArray()
+  const filter = argv.filter ? compileFilter(argv.filter) : null 
+  const opts = filter ? { filter } : { start, end, tags: argv.tag }
```

Comment:
There are a couple references to it down below as well. Like: 

https://github.com/tickbin/tickbin/pull/182/files#diff-b771a1bb827a2edfddb88b62f26c62d7R35